### PR TITLE
[QPE-537] Only supports auto number formatting

### PR DESCRIPTION
**Pull request by ahmed-Bazzara**
JIRA ticket
https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-537

### CODE REVIEW

File: src/paint.js [hunk 1]
```diff
@@ -3,14 +3,6 @@ import $ from 'jquery';
 import bullet from './vendor/bullet';
 
 //Function that returns 0 if the value passed is NaN or less than 0
```
Comment (kristofferlind) [inline src/paint.js]: remove the comment describing what the removed function does aswell

File: src/paint.js [hunk 2]
```diff
@@ -72,21 +65,21 @@ function createDataArray(hypercubeData, layout) {
     //check number of dimensions and build object based on the expressions available
     if (numMeasures == 1) {
       //use numDims to account for when chart does not have dimensions
-      dataObject[r]['measures'] = [validateBulletNums(dataPages[r][numDims].qText.replace(',', ''))];
+      dataObject[r]['measures'] = [dataPages[r][numDims].qNum];
```
Comment (kristofferlind) [inline src/paint.js]: Either make a named variable for dataPages[r] or rename r

File: src/vendor/bullet.js [hunk 3]
```diff
@@ -1,43 +1,60 @@
...
-          rangeMax = getRangeMax.call(this, d, i).slice().sort(d3.descending),
-          g = d3.select(this);
-
-      //set the x-axis scale based on the menu setting for universal vs independent
-      if (rangeMax==0) {
-            // Compute the new x-scale.
-            var x1 = d3.scale.linear()
-                .domain([0, Math.max(rangez[0], markerz[0], measurez[0])])
-                .range(reverse ? [width, 0] : [0, width]);
+      var rangez = ranges
```
Comment (muenva) [inline src/vendor/bullet.js]: `rangez` 😂

Comment (ahmed-Bazzara) [reply src/vendor/bullet.js]: Yup 😂

**Pull request merged**